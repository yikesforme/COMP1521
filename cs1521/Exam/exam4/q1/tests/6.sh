sh exe tests/s6.s
