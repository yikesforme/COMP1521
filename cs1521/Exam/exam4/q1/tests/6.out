s1 = @%&*@%
s1 = @%&*@%
s2 = @%&*@%
nv = 0
